### PR TITLE
Support 2nd set of UVs in BufferGeometry:fromGeometry()

### PR DESCRIPTION
**Pull request by WestLangley**
This is required for AO map and lightmap support.

### CODE REVIEW

**Comment by mrdoob** [comment]
Nice!

File: src/core/BufferGeometry.js [hunk 1]
```diff
@@ -213,6 +216,13 @@ THREE.BufferGeometry.prototype = {
 
 		}
 
+		if ( hasFaceVertexUv2 === true ) {
+
+			var uvs2 = new Float32Array( faces.length * 3 * 2 );
+			this.addAttribute( 'uv2', new THREE.BufferAttribute( uvs, 2 ) );
```
Comment (benaadams) [inline src/core/BufferGeometry.js]: Should be uvs2 rather than uvs:

```this.addAttribute( 'uv2', new THREE.BufferAttribute( uvs2, 2 ) );```

Comment (benaadams) [inline src/core/BufferGeometry.js]: PR #6374

Comment (WestLangley) [inline src/core/BufferGeometry.js]: @benaadams Ugh! Thank you.

**Pull request merged**